versions.yml

```diff
@@ -6,11 +6,25 @@ unifiedpush_apb: "1.0.1"
 keycloak_apb: "1.1.0"
 metrics_apb: "1.1.0"
 sync_app_apb: "1.1.0"
-js_sdk_app: "2.0.1"
-js_sdk_core: "2.0.1"
-js_sdk_auth: "2.0.0"
-js_sdk_push: "2.0.0"
-js_sdk_security: "2.0.0"
-js_sdk_cordova_plugin_aerogear_metrics: "2.0.0"
-js_sdk_cordova_plugin_aerogear_push: "2.0.0"
-js_sdk_cordova_plugin_aerogear_security: "2.0.0"
+
+# https://github.com/aerogear/aerogear-js-sdk
+js_sdk_app: "2.2.0"
+js_sdk_core: "2.2.0"
+js_sdk_auth: "2.2.0"
+js_sdk_push: "2.2.0"
+js_sdk_security: "2.2.0"
+js_sdk_cordova_plugin_aerogear_metrics: "2.2.0"
+js_sdk_cordova_plugin_aerogear_push: "2.2.0"
+js_sdk_cordova_plugin_aerogear_security: "2.2.0"
+
+# https://github.com/aerogear/ionic-showcase
+ionic-showcase: "0.3.0"
```

Comment:
it looks like this tag was not created.

Also I think we need to use underscore in the variables. Otherwise ansible will throw an error.

Reply:
We still testing that end to end. Going to let you know once tag will be created